packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -139,16 +147,30 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
       this.apiKey ? `&apiKey=${this.apiKey}` : ""
     ].join("");
 
-    const ohlcResponse = await this.networker.getJson(ohlcUrl);
+    // 2. Send requests.
+    const [ohlcResponse, priceResponse] = await Promise.all([
+      this.networker.getJson(ohlcUrl),
+      this.networker.getJson(priceUrl)
+    ]);
+
+    // 3. Check responses.
+    if (!priceResponse || !priceResponse.result || !priceResponse.result.price) {
+      throw new Error(`🚨Could not parse price result from url ${priceUrl}: ${JSON.stringify(priceResponse)}`);
+    }
```

Comment:
i saw you imported assert but didnt use it. you could here like:
```
assert(priceResponse && priceResponse.result && priceResponse.result.price,`🚨Could not parse price result from url ${priceUrl}: ${JSON.stringify(priceResponse)}`)
```

Reply:
Ha, yeah, I forgot to remove the `require`. I originally had exactly what you suggested, but @chrismaree rightfully pointed out that it's somewhat inconsistent with our other production code. I tend to agree that this might be better suited for a targeted refactor where we move to using asserts everywhere rather than having our error types/throwing mechanisms be different depending on the location.